pkg/reconciler/autoscaling/kpa/controller.go

```diff
@@ -58,16 +60,17 @@ func NewController(
 	endpointsInformer := endpointsinformer.Get(ctx)
 
 	c := &Reconciler{
-		Base:            reconciler.NewBase(ctx, controllerAgentName, cmw),
-		paLister:        paInformer.Lister(),
-		sksLister:       sksInformer.Lister(),
-		serviceLister:   serviceInformer.Lister(),
-		endpointsLister: endpointsInformer.Lister(),
-		kpaDeciders:     kpaDeciders,
-		metrics:         metrics,
+		Base:              reconciler.NewBase(ctx, controllerAgentName, cmw),
```

Comment:
I wonder if we should have a new base autoscaler reconciler...  🤔

Reply:
@mattmoor yes we should 😁. That's precisely what I'm working towards to remove code duplication between HPA and KPA paths. As this was buried even deeper though, I figured I'd rip this one out first, to keep the changeset manageable.